test/jdk/java/awt/print/PrinterJob/PrintNullString.java

```diff
@@ -21,308 +21,160 @@
-                       20, 120);
-        } */
-        try {
-             g.drawString(emptyIterator, 20, 140);
-             g.drawString("FAILURE: No IAE for empty iterator, int",
-                           20, 140);
-        } catch (IllegalArgumentException e) {
-          g.drawString("caught expected IAE for empty iterator, int",
-                        20, 140);
-        } /*catch (Exception e) {
-          g.drawString("FAILURE: unexpected exception for empty iterator, int",
-                       20, 140);
-        } */
-
-
-        // API 4: null & empty drawString(Iterator, float, int);
-        try {
-             g.drawString(nullIterator, 20.0f, 160.0f);
-             g.drawString("FAILURE: No NPE for null iterator, float", 20, 160);
-        } catch (NullPointerException e) {
-          g.drawString("caught expected NPE for null iterator, float", 20, 160);
-        } /*catch (Exception e) {
-          g.drawString("FAILURE: unexpected exception for null iterator, float",
-                        20, 160);
-        } */
-
-        try {
-             g.drawString(emptyIterator, 20, 180);
-             g.drawString("FAILURE: No IAE for empty iterator, float",
-                           20, 180);
-        } catch (IllegalArgumentException e) {
-          g.drawString("caught expected IAE for empty iterator, float",
-                        20, 180);
-        } /*catch (Exception e) {
-          g.drawString("FAILURE: unexpected exception for empty iterator, float",
-                       20, 180);
-        } */
-    }
-
-     public Dimension getPreferredSize() {
-        return new Dimension(450, 250);
-    }
- }
-
-}
-
-class Sysout
- {
-   private static TestDialog dialog;
-
-   public static void createDialogWithInstructions( String[] instructions )
-    {
-      dialog = new TestDialog( new Frame(), "Instructions" );
-      dialog.printInstructions( instructions );
-      dialog.show();
-      println( "Any messages for the tester will display here." );
-    }
-
-   public static void createDialog( )
-    {
-      dialog = new TestDialog( new Frame(), "Instructions" );
-      String[] defInstr = { "Instructions will appear here. ", "" } ;
-      dialog.printInstructions( defInstr );
-      dialog.show();
-      println( "Any messages for the tester will display here." );
-    }
-
-
-   public static void printInstructions( String[] instructions )
-    {
-      dialog.printInstructions( instructions );
+public class PrintNullString extends Frame {
+    private static final String INSTRUCTIONS =
+            "This test should print a page which contains the same\n" +
+            "text messages as in the test window on the screen\n" +
+            "The messages should contain only 'OK' and 'expected' messages\n" +
+            "If the page fails to print, but there were no exceptions\n" +
+            "then the problem is likely elsewhere (ie your printer)";
```

Comment:
```suggestion
            "This test should print a page which contains the same\n" +
            "text messages as in the test window on the screen.\n" +
            "\n" +
            "The messages should contain only 'OK' and 'expected' messages.\n" +
            "Press Pass if it's the case; otherwise press Fail.\n" +
            "\n" +
            "If the page fails to print, but there were no exceptions\n" +
            "then the problem is likely elsewhere (i.e. your printer)";
```